HISTORY.md

```diff
@@ -1,3 +1,8 @@
+1.10.0 / 2019-12-17
```

Comment:
Usually this is just the word "unreleased" until the release is made.

Reply:
Ofc, thx